netplan/cli/commands/apply.py

```diff
@@ -193,7 +211,7 @@ def command_apply(run_generate=True, sync=False, exit_on_error=True):  # pragma:
         # (re)start backends
         if restart_networkd:
             netplan_wpa = [os.path.basename(f) for f in glob.glob('/run/systemd/system/*.wants/netplan-wpa-*.service')]
-            netplan_ovs = [os.path.basename(f) for f in glob.glob('/run/systemd/system/*.wants/netpalan-ovs-*.service')]
+            netplan_ovs = [os.path.basename(f) for f in glob.glob('/run/systemd/system/*.wants/netplan-ovs-*.service')]
```

Comment:
wow, ok, missed that one! Guess this is the reason why `netplan apply` didn't quite work previously!

Reply:
No, I think it is more or less redundant to name the services explicitly here, as they are already triggered as a systemd dependencies via `systemd-networkd.service.wants` symlink, created by netplan `generate`.